docs: add note for throwSuggestions issue #1306 (#1425)
testing-library/testing-library-docs: docs/dom-testing-library/api-configuration.mdx

screen.getByTestId('foo', {suggest: false}) // will not throw a suggestion
```

> > **Note**

Reviewer: You can use a [Admonition](https://docusaurus.io/docs/markdown-features/admonitions) for this section, it should look better with it.

```suggestion
> :::note
```

screen.getByTestId('foo', {suggest: false}) // will not throw a suggestion
```

:::note

> When this option is enabled, the library may provide suggestions that lack an
> intuitive implementation, this typically occurs for
> [roles which can not be named](https://w3c.github.io/aria/#namefromprohibited),
> most notably paragraphs. For instance, if you attempt to use
> [`getByText`](queries/bytext.mdx), you may encounter the following error:

Reviewer: ```suggestion
When this option is enabled, it may provide suggestions that lack an
intuitive implementation. Typically this happens for
[roles which cannot be named](https://w3c.github.io/aria/#namefromprohibited),
most notably paragraphs. For instance, if you attempt to use
[`getByText`](queries/bytext.mdx), you may encounter the following error:
```

```
TestingLibraryElementError: A better query is available, try this:
    getByRole('paragraph')
```

> However, there is no direct way to query paragraphs using an object as the
> second parameter, such as in
> `getByRole('paragraph', { name: 'Hello World' })`.

Reviewer: ```suggestion
However, there is no direct way to query paragraphs using the config object parameter, such as in
`getByRole('paragraph', { name: 'Hello World' })`.
```

> To address this issue, you can leverage a custom function to validate the
> element's structure and suppress the error, see
> [#1306](https://github.com/testing-library/dom-testing-library/issues/1306)
> and the example below:

Reviewer: ```suggestion
To address this issue, you can leverage a custom function to validate the
element's structure, as shown in the example below.
More information can be found in the [GitHub issue](https://github.com/testing-library/dom-testing-library/issues/1306)
```
